### DISCUSSION TITLE
Increase SLURM compute node queue size when enable reconfigure is set as true

### DISCUSSION THREAD

**Discussion by irishwithag91** [GoogleCloudPlatform/cluster-toolkit#1533] [General]
Hi deployed a SLURM HPC cluster on Google cloud using hpc toolkit with "enable-reconfigure" set as true 
The compute node group currently stands at 10 nodes
Is there a way to increase the compute node group size to 20 nodes by making some configuration changes without having to re deploy and disturb the current setup?
Thanks

**Comment by tpdownes** [comment]
If you have set `enable_reconfigure: true` on your partition **and** you have installed the pre-requisite libraries as shown here (you might update URL to 5.7.4 instead of 5.7.3 though), you should be able to make simple configuration changes to your blueprint, then run `ghpc create` and `ghpc deploy`.

https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/d6a3ef45165c0c925b77b2e8eeddc9f69c35c22b/examples#hpc-slurmyaml-

Some users run into unexpected issues with this. Please let us know your experience.

**Comment by cboneti** [comment]
Hi, I will close this discussion since we haven't heard from you. Please feel free to reopen it if you are having issues. 
Cheers, 
Carlos
